### DISCUSSION TITLE
6주 스터디에 대한 Self-reflection

### DISCUSSION THREAD

**Discussion by briankim1027** [fusiondevkr/pl900#51] [1 Q&A]
제 후기는...
https://blog.naver.com/kimbrian/222489414473

수고많으셨고, 감사합니다.

**Comment by sesme100** [comment]
김혁님 블로그에서 퓨전 개발자 스터디 후기 남겨주신 블로그 포스트 확인 하였습니다!! 진심으로 감사드려요!!
